package.json

```diff
@@ -197,6 +197,7 @@
     "express-validator": "^6.14.2",
     "extract-zip": "^2.0.1",
     "fast-csv": "^4.3.6",
+    "htmlparser2": "^9.1.0",
```

Comment:
Added to comply with commit hooks. We already had it, as it is a depency of "html-webpack-plugin":

<img width="767" alt="into Reasons this module exists" src="https://github.com/user-attachments/assets/4b38b9bf-7fef-405e-8f0a-084463d820e1">